client/src/config.rs

```diff
@@ -119,6 +120,26 @@ impl Default for Configuration {
 }
 
 impl Configuration {
+    /// Key-pair used by default for demo purposes
+    #[allow(clippy::expect_used)]
+    fn default_keypair() -> KeyPair {
```

Comment:
I'd say these are placeholders, so `placeholder_account` and `placeholder_key_pair`.

Reply:
ok, I can do that